course_catalog/tasks.py

```diff
@@ -57,7 +57,7 @@ def get_edx_data(force_overwrite=False):
 
 
 @app.task
-def get_ocw_data(upload_to_s3=True):  # pylint:disable=too-many-locals
+def get_ocw_data(upload_to_s3=True):  # pylint:disable=too-many-locals,too-many-branches
```

Comment:
The disables are starting to pile up for this code and it's a lot of code for a task to begin with. I don't think it's good to mix a refactor with a feature change, so could you file a tech debt issue against this task implementation?

Reply:
https://github.com/mitodl/open-discussions/issues/1849